#! /bin/sh /usr/share/dpatch/dpatch-run
## 94_yaepg-1.3.38.dpatch by Thomas Gï¿½nther <tom@toms-cafe.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changes for VDR >= 1.3.38.

@DPATCH@
--- yaepg-0.0.2/yaepg.c
+++ yaepg-0.0.2/yaepg.c
@@ -1638,7 +1638,7 @@
 
 		case kBlue:
 			/* Can this be fixed to work with NTSC ? */
-			cDevice::PrimaryDevice()->GrabImageFile("img.jpg",
+			cDevice::PrimaryDevice()->GrabImageFile("/tmp/img.jpg",
 					true, 256, -1, -1);
 			state = osContinue;
 			break;
